server/src/logic.js

```diff
@@ -335,6 +335,99 @@ export const getHandlers = ({ models, creators: Creators, push }) => {
       eventLocations(event) {
         return event.getEventlocations();
       },
+      createEvent(_, args, ctx) {
+        const { name, details, sourceIdentifier, permalink, eventLocations, groupId } = args.event;
+        return getAuthenticatedUser(ctx).then(() =>
+          Group.findById(groupId).then((group) => {
+            if (!group) {
+              return Promise.reject(Error('Invalid group!'));
+            }
+            return Creators.event({
+              name,
+              details,
+              sourceIdentifier,
+              permalink,
+              group,
+            }).then((event) => {
+              if (eventLocations === undefined) return;
```

Comment:
have you verified this works? How is this different to `typeof eventLocations === 'undefined'` which I think was the recommended way to do this.

Reply:
I'll update it to preferred method